examples/microphone.py

```diff
@@ -0,0 +1,46 @@
+# Basic test for microphone.  This test should update the display when
+# Button A is pressed and a loud or quiet sound *is* heard, printing the
+# results. On Button B this test should update the display when a loud or
+# quiet sound *was* heard, printing the results. On shake this should print
+# the last sounds heard, you shoul try this test whilst making a loud sound 
+# and a quiet one before you shake.
+
+from microbit import *
+
+icons = {
+    SoundEvent.LOUD: Image.SQUARE,
+    SoundEvent.QUIET: Image.SQUARE_SMALL
+}
```

Comment:
Getting this error: TypeError: unsupported type for __hash__: 'MicroBitSoundEvent'

Reply:
Fixed in https://github.com/bbcmicrobit/micropython/pull/692/commits/78bd2418a5e2b22c538b233e3313806f2dc032e5 removed `icons` as with our test script